Items instantiated in header causes duplicate definitions and link errors

**Checklist (Please check before submitting)**

* [x] I reviewed the [Contributing Guide](https://github.com/nasa/LC/blob/main/CONTRIBUTING.md).
* [x] I performed a cursory search to see if the bug report is relevant, not redundant, nor in conflict with other tickets.

**Describe the bug**
The unit test header file `lc_test_utils.h` instantiates objects directly in the header file, which breaks if it is ever included in more than one C file.

**To Reproduce**
Build LC with unit tests enabled, get lots of linker errors:

```
usr/bin/ld: libcoverage-lc_internal-stubs.a(lc_test_utils.c.o):/home/joe/code/cfecfs/github/apps/lc/unit-test/utilities/lc_test_utils.h:39: multiple definition of `WDTable'; CMakeFiles/coverage-lc-lc_action-testrunner.dir/lc_action_tests.c.o:/home/joe/code/cfecfs/github/apps/lc/unit-test/utilities/lc_test_utils.h:39: first defined here
/usr/bin/ld: libcoverage-lc_internal-stubs.a(lc_test_utils.c.o):/home/joe/code/cfecfs/github/apps/lc/unit-test/utilities/lc_test_utils.h:40: multiple definition of `ADTable'; CMakeFiles/coverage-lc-lc_action-testrunner.dir/lc_action_tests.c.o:/home/joe/code/cfecfs/github/apps/lc/unit-test/utilities/lc_test_utils.h:40: first defined here
```

**Expected behavior**
Build should work?

**Code snips**
https://github.com/nasa/LC/blob/543cf6b8906cd976d33f14d3907567d6bc8c8cd9/unit-test/utilities/lc_test_utils.h#L38-L43

**System observed on:**
Ubuntu 

**Additional context**
I was just trying to build LC "out of the box" - not modified in any way - and it failed badly.  Not sure how this ever built or passed any validation testing with this the way it was.

**Reporter Info**
Joseph Hickey, Vantage Systems, Inc.

--- a/unit-test/utilities/lc_test_utils.c
+++ b/unit-test/utilities/lc_test_utils.c
@@ -34,6 +34,11 @@
 #include "utassert.h"
 #include "utstubs.h"
 
+LC_WDTEntry_t WDTable[LC_MAX_WATCHPOINTS];
+LC_ADTEntry_t ADTable[LC_MAX_ACTIONPOINTS];
+LC_WRTEntry_t WRTable[LC_MAX_WATCHPOINTS];
+LC_ARTEntry_t ARTable[LC_MAX_ACTIONPOINTS];
+
 #define UT_MAX_SENDEVENT_DEPTH 4
 CFE_EVS_SendEvent_context_t    context_CFE_EVS_SendEvent[UT_MAX_SENDEVENT_DEPTH];
 CFE_ES_WriteToSysLog_context_t context_CFE_ES_WriteToSysLog;

--- a/unit-test/utilities/lc_test_utils.h
+++ b/unit-test/utilities/lc_test_utils.h
@@ -36,10 +36,10 @@ extern LC_AppData_t  LC_AppData;
 extern LC_OperData_t LC_OperData;
 
 /* Global table variables for table pointers contained in LC_OperData */
-LC_WDTEntry_t WDTable[LC_MAX_WATCHPOINTS];
-LC_ADTEntry_t ADTable[LC_MAX_ACTIONPOINTS];
-LC_WRTEntry_t WRTable[LC_MAX_WATCHPOINTS];
-LC_ARTEntry_t ARTable[LC_MAX_ACTIONPOINTS];
+extern LC_WDTEntry_t WDTable[LC_MAX_WATCHPOINTS];
+extern LC_ADTEntry_t ADTable[LC_MAX_ACTIONPOINTS];
+extern LC_WRTEntry_t WRTable[LC_MAX_WATCHPOINTS];
+extern LC_ARTEntry_t ARTable[LC_MAX_ACTIONPOINTS];
 
 /*
  * Global context structures

